### PR TITLE
Some UI fixes

### PR DESCRIPTION
**Pull request by CPatchane**

### CODE REVIEW

File: cozy-theme/nav.html [hunk 1]
```diff
@@ -15,7 +15,8 @@
 
           {%- block site_name %}
             <a class="navbar-brand" href="https://docs.cozy.io/">{{ config.site_name }}</a>
-            <a class="navbar-brand" href="https://docs.cozy.io/en">en</a>&nbsp;<a class="navbar-brand" href="https://docs.cozy.io/fr/index_fr">fr</a>
+            <a class="navbar-brand" href="https://docs.cozy.io/en">en</a>
+            <a class="navbar-brand" href="https://docs.cozy.io/fr/index_fr">fr</a>
```
Comment (enguerran) [inline cozy-theme/nav.html]: Why did you remove the blank space?

File: cozy-theme/nav.html [hunk 2]
```diff
@@ -67,7 +68,7 @@
                         </a>
                     </li>
                 {%- endif %}
-              {%- endblock %}
+              {%- endblock %} -->
```
Comment (enguerran) [inline cozy-theme/nav.html]: Do not comment code, git is our memory, if we need it back, we can get it back from git revisions.
Future ourselves will never understand why this is commented out, help them by making clear action. And if you really want to remove this code but with revert possibility, just remove it in one single commit with a really explaining comment.

File: src/extra.css [hunk 3]
```diff
@@ -130,7 +130,7 @@ a:hover, a:active {
   font-size: 1.2em;
 }
 .home-action {
-  width: 45%;
+  width: calc(50% - 2em);
```
Comment (enguerran) [inline src/extra.css]: What about browser than does not support `calc()`?

File: cozy-theme/nav.html (hunk 1 above)
Comment (enguerran) [reply cozy-theme/nav.html]: I don't say, here, to not remove it, I say: could you explain yourself in the Pull Request comments?

File: cozy-theme/nav.html (hunk 2 above)
Comment (CPatchane) [reply cozy-theme/nav.html]: I wanted to remove this code while explaining it can be possible for later use. So didn't want to remove it completely.

File: src/extra.css (hunk 3 above)
Comment (CPatchane) [reply src/extra.css]: All except Opera Mini according to https://caniuse.com/#feat=calc

File: cozy-theme/nav.html (hunk 1 above)
Comment (CPatchane) [reply cozy-theme/nav.html]: I don't know, it seems to be unnecessary and unused. I think @clochix could know if it's important or not.

File: cozy-theme/nav.html (hunk 2 above)
Comment (enguerran) [reply cozy-theme/nav.html]: Ne commente pas le code, git est notre mémoire, si nous avons finalement besoin de ce code, nous pourrons le récupérer grâce aux révisions git.
Nos moi futurs ne comprendront jamais pourquoi ce code est commenté, aidons les en faisant des actions claires. Et si tu penses que nous devrions supprimer ce code mais que nous pourrions en avoir besoin plus tard, retire-le dans un commit distinct avec un commentaire clair.

File: src/extra.css (hunk 3 above)
Comment (enguerran) [reply src/extra.css]: o_O wow, I thought it was less supported.

**Comment by clochix** [comment]
I really have no time to review this one, so let’s merge and see.
Thanks @CPatchane

**Comment by clochix** [comment]
Unfortunately, navigation bar is still present and broken :-(
